### PR TITLE
[i18n] Front-end

### PR DESCRIPTION
**Pull request by soupette**
Waiting for #9535 to be merged.

## How to test

1. In one terminal window clone the monorepo checkout the branch corresponding to #9535 PR's branch && start the `getstarted` app.
2. In another terminal checkout the `i18n/rbac-rework-front` branch and start the webpack `devServer`

### CODE REVIEW

File: packages/strapi-plugin-content-manager/admin/src/containers/ListViewLayoutManager/index.js [hunk 1]
```diff
@@ -1,31 +1,40 @@
...
+  const setQueryRef = useRef(setQuery);
 
   useEffect(() => {
     dispatch(setLayout(layout.contentType));
   }, [dispatch, layout]);
 
   useEffect(() => {
-    if (initialParams) {
-      setQuery(initialParams);
+    // We need to keep the search when reloading the page
```
[the remark this answers is not in the document]
Comment (mfrachet) [reply packages/strapi-plugin-content-manager/admin/src/containers/ListViewLayoutManager/index.js:21]: What would be the pros of approaching this computation from the menu side?

File: packages/strapi-helper-plugin/lib/src/utils/contentManagementUtilRemoveFieldsFromData.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ import { getType, getOtherInfos } from './getAttributeInfos';
 
 const defaultFields = ['created_by', 'updated_by', 'published_at', 'id', '_id'];
 
-const removeFieldsFromClonedData = (
+const contentManagementUtilRemoveFieldsFromData = (
```
[the remark this answers is not in the document]
Comment (mfrachet) [reply packages/strapi-helper-plugin/lib/src/utils/contentManagementUtilRemoveFieldsFromData.js:6]: This one is used both in CM and I18N. I think it's the reason why it's in helper

File: packages/strapi-plugin-content-manager/admin/src/containers/ListViewLayoutManager/index.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (soupette) [reply packages/strapi-plugin-content-manager/admin/src/containers/ListViewLayoutManager/index.js:21]: We will have to same way of managing the single types links and the collection types links.

**Review by mfrachet — APPROVED** [review]
In the actual situation (waiting for the backend folks), I think this one can get merged. We'll then be able to create branches from the i18n one and fix stuff directly.

WYT?

**Pull request merged**